source/server/server.cc

```diff
@@ -903,7 +902,7 @@ RunHelper::RunHelper(Instance& instance, const Options& options, Event::Dispatch
   // If there is no global limit to the number of active connections, warn on startup.
   if (!overload_manager.getThreadLocalOverloadState().isResourceMonitorEnabled(
           Server::OverloadProactiveResourceName::GlobalDownstreamMaxConnections) &&
-      !instance.runtime().snapshot().get(Network::TcpListenerImpl::GlobalMaxCxRuntimeKey)) {
+      !instance.runtime().snapshot().get("overload.global_downstream_max_connections")) {
```

Comment:
Could we perhaps relocate this string to somewhere that would allow us to refer to it by the variable name, instead of repeating the string?

Reply:
ack